README.md

```diff
@@ -38,6 +38,28 @@ A: No. You can, but running `bundle exec zeus` instead of `zeus` adds precious s
 It is common to see tests running twice when starting out with Zeus. If you see your tests/specs running twice, you should try disabling `require 'rspec/autotest'` and `require 'rspec/autorun'` (for RSpec), or `require 'minitest/autorun'` (for Minitest). (see [#134](https://github.com/burke/zeus/issues/134) for more information).
 
 
+## Set up 
+
+In your Rails app's directory initialize zeus:
+
+    zeus init
+
+which will create two files that should be added to your `.gitignore` (or globally in `~/.gitignore`)
+
+    custom_plan.rb
+    zeus.json
+
+Force the test environment: In `spec/spec_helper.rb`, change:
```

Comment:
Good point. In fact `test/test_helper.rb` should be the default, with `spec/spec_helper.rb` included as an addendum. So changed.